amd/src/cv_launch.js

```diff
@@ -71,7 +77,7 @@ define(['jquery', 'core/str'], function($, str) {
                     checkDVClosed: function(cvWindow) {
                         var that = this;
                         if (cvWindow.closed) {
-                            window.location = window.location;
+                            window.location = window.location + '';
```

Comment:
🤨

Reply:
In 3.9 grunt complains about the variable being assigned to itself, so the JS wouldn't compile. It didn't complain in earlier versions. This is a bit of a hack to get around that. I essentially copied what John did to get around it in the TFS plugin.